### PR TITLE
[SMTChecker] Support shifts

### PR DESCRIPTION
**Pull request by leonardoalt**
Ref https://github.com/ethereum/solidity/issues/9043

### CODE REVIEW

File: libsmtutil/SolverInterface.h [hunk 1]
```diff
@@ -299,15 +302,30 @@ class Expression
 		auto bvSort = _a.sort;
 		return Expression("bvand", {std::move(_a), std::move(_b)}, bvSort);
 	}
+	friend Expression operator|(Expression _a, Expression _b)
```
Comment (leonardoalt) [inline libsmtutil/SolverInterface.h:305]: This was just moved up here for consistency with the map above.

**Comment by leonardoalt** [comment]
This is ready for review.
The tests were copied from the semantic tests, where each call was transformed into an assertion (and a false version).

**Review by chriseth — DISMISSED** [review]

**Comment by chriseth** [comment]
I guess the expectations need to be adjusted anyway after the previous merge.

**Comment by leonardoalt** [comment]
Yep will do

**Review by chriseth — APPROVED** [review]

**Pull request merged**